css/documents.css

```diff
@@ -0,0 +1,153 @@
+.tna-breadcrumb .breadcrumbs {
```

Comment:
This will be added to the feature 'removing bxslider', which will be added to the next release.